components/ui-api-layer/internal/domain/servicecatalog/usage_kind_service_test.go

```diff
@@ -48,30 +48,26 @@ func TestUsageKindService_List_Empty(t *testing.T) {
 	require.NoError(t, err)
 
 	// THEN
-	assert.Len(t, result, 0)
+	assert.Empty(t, result)
```

Comment:
TestUsageKindService_List_Success does not test too much, there is the only assertion about the length of a returned slice.

Reply:
I changed `TestUsageKindService_List_Success` assertion about the length of a returned slice. Now i check result using assert.Equal().